src/charts/stackedArea/stackedAreaChart.js

```diff
@@ -0,0 +1,13 @@
+import stackedAreaChart from 'britecharts/dist/umd/stackedArea.min';
```

Comment:
are you sure you want to use the minimized code here? it'll prevent users from getting decent stack traces. I think you should use the ESM version so that you can get tree-shaking. Then when the client app creates its bundle, everything will be minified

Reply:
Right now Britecharts doesn't expose ESM versions, but we will be working on adding them soon!